conf/evolutions/default/1.sql

```diff
@@ -0,0 +1,71 @@
+# --- !Ups
+
+create table "movie" (
+	"id" bigserial primary key,
+	"title" varchar not null,
+	"description" text not null,
+	"release-date" date not null,
+	"country" varchar not null,
+	"language" varchar(2) not null
+);
+
+create table "director" (
+	"id" bigserial primary key,
+	"first-name" varchar not null,
+	"last-name" varchar not null,
+	"birth-date" date not null,
+	"nationality" varchar not null,
+	"height" int not null,
```

Comment:
Validation for height is implemented in form mapping, but not in database column definition. Defining proper constants  on the data storage side (if that is possible (in the case of relational databases - VERY POSSIBLE)) is as important as doing so in front-end, if not even more important, because it guarantees data model correctness.